osbs/repo_utils.py

```diff
@@ -60,10 +60,64 @@ def __init__(self, dir_path='', file_name=REPO_CONFIG_FILE):
             with open(file_path) as f:
                 self.container = (yaml.load(f) or {})
 
+        modules = self.container.get('compose', {}).get('modules', [])
+        self.container_module_specs = [ModuleSpec.from_text(module) for module in modules]
```

Comment:
we should gather all errors first (for all modules) and raise error after, that way user will see all wrong module specs at once, and not 1 by 1